sklearn/model_selection/tests/test_split.py

```diff
@@ -991,37 +958,6 @@ def test_repeated_cv_repr(RepeatedCV):
     assert repeated_cv_repr == repr(repeated_cv)
 
 
-def test_repeated_kfold_determinstic_split():
```

Comment:
replaced by `test_random_state_reproducibility` and `test_repeated_folds_are_different`